src/pages/IDO/index.tsx

```diff
@@ -0,0 +1,186 @@
+import { darken } from 'polished'
+import { CardSection, DataCard, CardNoise, CardBGImage } from '../../components/earn/styled'
+
+import { IDO_LIST } from '../../constants/idos'
+
+import React from 'react'
+import { AutoColumn } from '../../components/Column'
+import styled from 'styled-components'
+import { TYPE, ExternalLink } from '../../theme'
+
+import { RowBetween } from '../../components/Row'
+
+import { useTranslation } from 'react-i18next'
+
+const PageWrapper = styled(AutoColumn)``
+
+const TopSection = styled(AutoColumn)`
+  max-width: 640px;
+  width: 100%;
+`
+
+const IDOCard = styled(DataCard)`
+  background: radial-gradient(76.02% 75.41% at 1.84% 0%, #27ae60 0%, #000000 100%);
+  overflow: hidden;
+`
+
+const WrapSmall = styled(RowBetween)`
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    flex-wrap: wrap;
+
+  `};
+`
+
+/* the idea is to format the partner logos with this one */
+const PngIcon = styled.div`
+  transition: transform 0.3s ease;
+  :hover {
+    transform: rotate(-5deg);
+  }
+`
+const activeClassName = 'ACTIVE'
+
+const StyledExternalLink = styled(ExternalLink).attrs({
+  activeClassName
+})<{ isActive?: boolean }>`
+  ${({ theme }) => theme.flexRowNoWrap}
+  align-items: left;
+  border-radius: 3rem;
+  outline: none;
+  cursor: pointer;
+  text-decoration: none;
+  color: ${({ theme }) => theme.text2};
+  font-size: 1rem;
+  width: fit-content;
+  margin: 0 12px;
+  font-weight: 500;
+
+  &.${activeClassName} {
+    border-radius: 12px;
+    font-weight: 600;
+    color: ${({ theme }) => theme.text1};
+  }
+
+  :hover,
+  :focus {
+    text-decoration: none;
+    color: ${({ theme }) => darken(0.1, theme.text1)};
+  }
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+      display: none;
+`}
+`
+const IDOs = styled.div`
+  padding: 0.75rem 1rem;
+  width: 100%;
+  margin-top: 1rem;
+  border-radius: 12px;
+  display: grid;
+  grid-template-columns: 60px 1fr 100px;
+  align-items: center;
+  text-align: left;
+  outline: none;
+  cursor: pointer;
+  color: ${({ theme }) => theme.text1};
+  text-decoration: none;
+  background-color: ${({ theme }) => theme.bg1};
+  &:focus {
+    background-color: ${({ theme }) => darken(0.05, theme.bg1)};
+  }
+  &:hover {
+    background-color: ${({ theme }) => darken(0.05, theme.bg1)};
+  }
+`
+
+const EmptyProposals = styled.div`
+  border: 1px solid ${({ theme }) => theme.text4};
+  padding: 16px 12px;
+  border-radius: 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const IDO_STATUS_UPCOMING = 'Upcoming'
```

Comment:
These should be exports from the `../../constants/idos` file referenced in the config there and here so devs won't fat finger ex. "Upcoming" to "Upconing" and break the filtering logic

Reply:
moved them to the constants/idos.tsx and imported them in IDO/index.tsx